rasa/nlu/utils/spacy_utils.py

```diff
@@ -85,62 +82,19 @@ def cache_key(
         cls, component_meta: Dict[Text, Any], model_metadata: "Metadata"
     ) -> Optional[Text]:
 
-        spacy_model_name = cls._check_model_fallback(
-            component_meta.get("model"), model_metadata.language, warn=False
-        )
+        if not component_meta.get("model"):
+            raise InvalidModelError(
+                f"Missing model configuration for `SpacyNLP` in `config.yml`.\n"
+                f"You must pass a model to the `SpacyNLP` component explicitly.\n"
+                f"For example:\n"
+                f"- name: SpacyNLP\n"
+                f"  model: en_core_web_md\n"
+                f"More informaton can be found on {DOCS_URL_COMPONENTS}#spacynlp"
+            )
 
```

Comment:
Seems good!
Just one question though: shouldn't we also write a test for this error message?

Reply:
This is handled by the `test_model_raises_error_no_model_given`-test.

Reply:
Ah... you mean you want to test the contents of the message? We could go there but that feels like a bit of overkill.

Reply:
I checked and `test_model_raises_error_no_model_given` covers the error in `load_model` and not this one. Am I missing something?
<img width="1045" alt="error" src="https://user-images.githubusercontent.com/45267439/125294901-fa252c00-e324-11eb-9bf7-e76832bb5906.png">

Reply:
Yep. I just checked and fixed it. There was actually a small bug there where it gave the wrong error message. Just fixed it by changing the test and making a fix. 

Good call on your end!